pkg/k8s/init.go

```diff
@@ -34,22 +34,21 @@ func Init() error {
 	if nodeName := os.Getenv(EnvNodeNameSpec); nodeName != "" {
 		// Use of the environment variable overwrites the node-name
 		// automatically derived
-		nodeaddress.SetName(nodeName)
+		node.SetName(nodeName)
 
 		k8sNode, err := GetNode(Client(), nodeName)
 		if err != nil {
 			return fmt.Errorf("unable to retrieve k8s node information: %s", err)
 		}
 
-		node := ParseNode(k8sNode)
+		n := ParseNode(k8sNode)
+		log.Infof("Retrieved node's %s information from kubernetes", n.Name)
```

Comment:
`log.WithField(logfields.NodeName, n.Name).Info("Retrieved node information from kubernetes")` to be consistent with our logrus usage.

Reply:
fixed